src/hooks/use-sirs-token.hook.ts

```diff
@@ -0,0 +1,11 @@
+import { useTokenSelector } from '../store/wallet/wallet-selectors';
+import { LIQUIDITY_BAKING_LP_SLUG } from '../token/data/token-slugs';
+import { isDefined } from '../utils/is-defined';
+
+export const useSirsToken = () => {
+  const sirsToken = useTokenSelector(LIQUIDITY_BAKING_LP_SLUG);
+
+  const isSirsBalance = isDefined(sirsToken) && Number(sirsToken.balance) > 0;
+
+  return { isSirsBalance, sirsToken };
+};
```

Comment:
rename hook and properties returned by the hook.  

what I don't like:  
1. useSirs**Token** do not return TokenInterface (mb could be renamed to useSirsInfo)
2. use**Sirs**Token returns { is**Sirs**Balance, **sirs**Token } (duplication)
3. isSirsBalance do not represent what it is actually (mb isPositiveBalance)